### ISSUE TITLE
v0.2.12.2: continuous 'start silence' messages in log

### ISSUE THREAD

**Issue by pwt** [philippe44/AirConnect#184]
Starting with airupnp-arm v0.2.12.2, when streaming, I'm seeing continuous logging of messages of the form:
`_buffer_get_frame:944 [0xb1014d18]: start silence (late -960 ms) [W:1285 R:1285]`

The stream is playing fine, however. 

v0.2.12.0 (the most recent previous version of airupnp-arm) does not suffer from this issue.

Logging is configured as follows:
```
<main_log>info</main_log>
<upnp_log>warn</upnp_log>
<util_log>warn</util_log>
<raop_log>warn</raop_log>
```

Any ideas?

**Comment by philippe44** [comment]
I'm not able to reproduce that, can you tell me more about your parameters

**Comment by pwt** [comment]
Sure, and thanks for looking at this.

Here's the config.xml file:
```
<?xml version="1.0"?>
<airupnp>
     <common>
         <enabled>1</enabled>
         <max_volume>100</max_volume>
         <codec>flc</codec>
         <metadata>1</metadata>
         <artwork>https://raw.githubusercontent.com/pwt/docker-airconnect-arm/master/airconnect-logo.png</artwork>
         <latency>1000:1000:f</latency>
         <drift>1</drift>
     </common>
     <main_log>info</main_log>
     <upnp_log>warn</upnp_log>
     <util_log>warn</util_log>
     <raop_log>warn</raop_log>
     <log_limit>2</log_limit>
</airupnp>
```

And here's the command line:
`./airupnp-arm -x config.xml -Z -m PLAYBASE,BEAM,One -n S6,S16`

For the same audio stream, running with v0.2.12.0 produces only the usual routine logging messages, running with v0.2.13.0 produces a continuous stream of 'start silence' messages. The stream plays fine in both cases.

**Comment by pwt** [comment]
In case it's relevant, the XXX in the `(late -XXX ms)` part of the logging message is always around 950-980ms.

**Comment by philippe44** [comment]
Found the problem. I backported modifications from my squeezeAMP/squeezelite-esp32 projects and I should not have

**Comment by pwt** [comment]
Confirmed fixed in v0.2.13.1. Thanks @philippe44!
